portal/apps/org.wso2.carbon.iam.admin-portal.app/src/main/pages/users/add.js

```diff
@@ -94,3 +125,30 @@ function userRegistration(claimMap, credentialMap, domain) {
         };
     }
 }
+
+function userRegistrationWithAskPassword(claimMap, credentialMap, domain){
```

Comment:
Why use a new 'userRegistrationWithAskPassword' method? Cant we use the same 'userRegistration' method used in the other cases?